### ISSUE TITLE
Crush while starting as module without parameters

### ISSUE THREAD

**Issue by antocuni** [pdbpp/pdbpp#60]
Originally reported by: **Eugene (Bitbucket: [eugene_bright](https://bitbucket.org/eugene_bright), GitHub: Unknown)**

----------------------------------------

If pdbpp is started as module without patameters, it crashes down.
```
python3 -m pdb 
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.5/dist-packages/_pdbpp_path_hack/pdb.py", line 5, in <module>
    os.path.dirname(os.path.dirname(__file__)), 'pdb.py'), 'exec'))
  File "/usr/local/lib/python3.5/dist-packages/pdb.py", line 1096, in <module>
    pdb.main()
  File "/usr/lib/python3.5/pdb.py", line 1632, in main
    print(_usage)
NameError: name '_usage' is not defined
```
If started with python script name as a parameter, when it works.

Version: 0.8.3
```
python3 -c "import pdb; print(pdb.get_version())"
0.8.3
```

----------------------------------------
- Bitbucket: https://bitbucket.org/antocuni/pdb/issue/60

**Comment by antocuni** [comment]
*Original comment by* **Семён Марьясин (Bitbucket: [MarSoftWare](https://bitbucket.org/MarSoftWare), GitHub: Unknown)**:

----------------------------------------

Looks like it is not so trivial. `_usage` is defined as original `pdb`'s global variable, and it is still available. But at the same time it is not available for `pdb`'s methods, including `main()`. Most likely it is because of the way we import it in `import_from_stdlib()`; on my system it uses read-compile-exec approach.

**Comment by MarSoft** [comment]
It seems that this bug was fixed in `pdbpp==0.9.3`. I think it can be closed now.

**Comment by blueyed** [comment]
Yes, thanks.
